app/models/miq_websocket_worker.rb

```diff
@@ -0,0 +1,34 @@
+class MiqWebsocketWorker < MiqWorker
+  require_nested :Runner
+
+  REQUIRED_ROLE = 'websocket'
```

Comment:
websocket role name seems awfully generic.  Is this role expected to handle any websocket stuff now and in the future?  If not, perhaps this role should be named such that it's remote console specific?